### PR TITLE
Updated Database.md

### PR DESCRIPTION
**Pull request by apu52**
Updated Portfolio box & Flatiron school student offer

# Issue Number : 
.
.
.
# Modified File Path : 
.
.
.
# Description : 
.
.
.
# Mentions : 
@ShreyamMaity

### CODE REVIEW

File: Database/database.md [hunk 1]
```diff
@@ -58,7 +58,7 @@
 |[Gurobi](https://www.gurobi.com/academia/academic-program-and-licenses/)|Student License|Business Problem Solving|
 |[Shapr3d](https://www.shapr3d.com/education)|Student License 1 Year|3D modeling App|
 |[Gliffy](https://help.gliffy.com/online/Content/GliffyOnline/free_for_students.htm)|Free account for active students|Diagram Drawing|
-|[Portfolio Box Pro](https://)|Student License|Portfolio creation|
+|[Portfolio Box Pro](https://https://www.portfoliobox.net/students)|Student License|Portfolio creation|
```
Comment (ShreyamMaity) [inline Database/database.md:61]: remove that extra https://

**Review by ShreyamMaity — CHANGES_REQUESTED** [review]

**Comment by ShreyamMaity** [comment]
@apu52

File: Database/database.md (hunk 1 above)
Comment (apu52) [inline Database/database.md:61]: I Removed it

**Review by apu52 — COMMENTED** [review]
removed

**Pull request merged**